packages/jest-worker/src/workers/NodeThreadsWorker.ts

```diff
@@ -154,11 +174,11 @@ export default class ExperimentalWorker implements WorkerInterface {
     return this._options.workerId;
   }
 
-  getStdout() {
-    return this._worker.stdout;
+  getStdout(): NodeJS.ReadableStream | null {
```

Comment:
This is the type defined in the interface, no? Does TS requires you to add this explicitly?

Reply:
Yeah, otherwise I'm getting this error:

[ts] Return type of public method from exported class has or is using name 'MergedStream' from external module "/Users/rubennorte/gitrepos/github/jest/node_modules/@types/merge-stream/index" but cannot be named.